form_urlencoded/src/lib.rs

```diff
@@ -345,3 +327,20 @@ fn append_pair(
 fn append_encoded(s: &str, string: &mut String, encoding: EncodingOverride) {
     string.extend(byte_serialize(&query_encoding::encode(encoding, s.into())))
 }
+
+fn decode_utf8_lossy(input: Cow<[u8]>) -> Cow<str> {
```

Comment:
Why move this from the encoding module?

Reply:
I think this is just a fragment of me thinking that this was the only functionality form_urlencoded uses from that module and moving it out, to then move query_encoding back into the url crate.